docs/section5/Rule5-45.md

```diff
@@ -0,0 +1,41 @@
+
+# Envelope - Rule 5-45  
+
+**Rule ID:** 5-45  
+**Rule Description:** The  infiltration schedules are the same in the proposed RMR as in the baseline RMR.  
+**Rule Assertion:** B-RMR hourly fractions of infiltration:infiltration_multiplier_schedule_name = P-RMR hourly fractions of infiltration:infiltration_multiplier_schedule_name.  
```

Comment:
Rule assertion should be that the schedules are the same, not that the names of the schedules are the same

Reply:
Corrected to infiltration.multiplier_schedule.